test/extended/util/annotate/rules.go

```diff
@@ -66,6 +66,11 @@ var (
 
 			// https://bugzilla.redhat.com/show_bug.cgi?id=2004074
 			`\[sig-network-edge\]\[Feature:Idling\] Unidling should work with TCP \(while idling\)`,
+
+			// TODO: to facilitate v.14 rebase, must be removed after the rebase PR lands
+			`\[sig-auth\] ServiceAccounts should allow opting out of API token automount`,
+			`\[sig-instrumentation\] Events API should ensure that an event can be fetched, patched, deleted, and listed`,
```

Comment:
See bug https://bugzilla.redhat.com/show_bug.cgi?id=2081084, to make sure we don't ship without fixing this. 
I think we see this test failing because origin/test (from previous version v1.23) is running against an apiserver (v1.24 with rebase PR)
So hopefully, when re re-vendor v1.24 into openshift/origin it should resolve, otherwise we will need to fix it before we ship 4.11.0

Reply:
Please locate the fix and ideally fix it here.  We have previously created blocker+ bugs, which were later deferred.  Let's do better.

Reply:
this relates to https://github.com/kubernetes/kubernetes/pull/108047. so the old test in openshift/origin is running against a new apiserver.